geo/Geoserver.py

```diff
@@ -728,11 +728,6 @@ def create_layergroup(
                 raise Exception(
                     f"Format not supported. Acceptable formats are : {supported_formats}"
                 )
-            # check if it already exist in Geoserver
-            if self.get_layergroup(name) is not None:
-                raise Exception(
-                    f"Layergroup: {name} already exist in Geoserver instance"
-                )
```

Comment:
I'm not sure whether it is smart to remove the check entirely, because this will cause issues for any code that assumes it will get an error if the layergroup exists already and will overwrite the existing layer.

I would suggest to change it, so that it checks for an exception in get_layergroup, like (I don't know if this is the most pythonic solution, but it works):

```
            # check if it already exist in Geoserver
            try: 
                existing_layer_group = self.get_layergroup(name)
            except:
                existing_layer_group = None

            if existing_layer_group is not None:
                raise Exception(
                    f"Layergroup: {name} already exists in Geoserver instance"
                )
```

Reply:
Yes, I think it is the correct way of fixing for now. Can you please update this PR with above suggestion? @julescarpentier